CodeStarterCamp_Week1/main.swift

```diff
@@ -1,12 +1,16 @@
-//
-//  main.swift
-//  CodeStarterCamp_Week1
-//
-//  Created by yagom.
-//  Copyright © yagom academy. All rights reserved.
-//
 
-import Foundation
+func iceCreamHead (number: Int) {
```

Comment:
```suggestion
func iceCreamHead(number: Int) {
```
함수명과 파라미터 사이에는 공백을 없애주시면 좋을 거 같습니다 ㅎㅎ

Reply:
네 확인했습니다!